darts/timeseries.py

```diff
@@ -1545,40 +1546,71 @@ def data_array(self, copy=True) -> xr.DataArray:
         """
         return self._xa.copy() if copy else self._xa
 
-    def pd_series(self, copy=True) -> pd.Series:
+    def to_series(
+        self,
+        copy: bool = True,
+        backend: Literal["pandas", "polars", "pyarrow"] = "pandas",
+    ):
         """
-        Return a Pandas Series representation of this univariate deterministic time series.
+        Return a Series representation of this univariate deterministic time series.
 
         Works only for univariate series that are deterministic (i.e., made of 1 sample).
 
         Parameters
         ----------
         copy
             Whether to return a copy of the series. Leave it to True unless you know what you are doing.
+        backend
+            The library used for the Series.
 
         Returns
         -------
-        pandas.Series
-            A Pandas Series representation of this univariate time series.
+            A Series representation of this univariate time series.
         """
         self._assert_univariate()
         self._assert_deterministic()
+
+        if backend != "pandas":
+            return self.to_dataframe(copy=copy, backend=backend)
+
+        data = self._xa[:, 0, 0].values
+        index = self._time_index
+        name = self.components[0]
+
         if copy:
-            return pd.Series(
-                self._xa[:, 0, 0].values.copy(),
-                index=self._time_index.copy(),
-                name=self.components[0],
-            )
-        else:
-            return pd.Series(
-                self._xa[:, 0, 0].values,
-                index=self._time_index,
-                name=self.components[0],
-            )
+            data = data.copy()
+            index = index.copy()
 
-    def pd_dataframe(self, copy=True, suppress_warnings=False) -> pd.DataFrame:
+        return pd.Series(data=data, index=index, name=name)
+
+    def pd_series(self, copy: bool = True) -> pd.Series:
         """
-        Return a Pandas DataFrame representation of this time series.
+        Return a Pandas Series representation of this univariate deterministic time series.
+
+        Works only for univariate series that are deterministic (i.e., made of 1 sample).
+
+        Parameters
+        ----------
+        copy
+            Whether to return a copy of the series. Leave it to True unless you know what you are doing.
+
+        Returns
+        -------
+        pandas.Series
+            A Pandas Series representation of this univariate time series.
+        """
+        logger.warning("`pd_series` is deprecated, please use `to_series` instead")
+        return self.to_series(copy=copy, backend="pandas")
+
+    def to_dataframe(
+        self,
+        copy: bool = True,
+        backend: Literal["pandas", "polars", "pyarrow"] = "pandas",
+        time_as_index: bool = False,
```

Comment:
I'd prefer to have `time_as_index=True` as default, since our users are used to `pd_dataframe()`, and that was the default there